src/traffic_layout/file_system.cc

```diff
@@ -41,17 +41,9 @@ static std::string remove_path;
 CopyStyle copy_style;
 
 // list of all executables of traffic server
-std::set<std::string> const executables = {"traffic_crashlog",
-                                           "traffic_ctl",
-                                           "traffic_layout",
-                                           "traffic_logcat",
-                                           "traffic_logstats",
-                                           "traffic_server",
-                                           "traffic_top",
-                                           "traffic_via",
-                                           "trafficserver",
-                                           "tspush",
-                                           "tsxs"};
+std::set<std::string> const executables = {"traffic_crashlog", "traffic_ctl",    "traffic_layout", "traffic_logcat",
+                                           "traffic_logstats", "traffic_server", "traffic_top",    "traffic_via",
+                                           "trafficserver",    "tspush"};
```

Comment:
put a comma after "tspush" to fix up formatting.

Reply:
The "improvements" with the extra comma are pretty insignificant, it adds an empty line after the '{', and pushes things to the left. 

```
+std::set<std::string> const executables = {
+  "traffic_crashlog", "traffic_ctl", "traffic_layout", "traffic_logcat", "traffic_logstats",
+  "traffic_server",   "traffic_top", "traffic_via",    "trafficserver",  "tspush",
+};
```